doc/openfortivpn.1.in

```diff
@@ -68,7 +69,11 @@ Specify a custom config file (default: @SYSCONFDIR@/openfortivpn/config).
 VPN account username.
 .TP
 \fB\-p \fI<pass>\fR, \fB\-\-password=\fI<pass>\fR
-VPN account password.
+VPN account password in plain text. For a secure alternative, use pinentry.
```

Comment:
I would argue `pinentry` is a secure alternative to the `password` **config file option**. Alternatives to the `--password` **command line option** include both `pinentry` (persistent with macOS keychain) and letting `openfortivpn` prompt for the password (not persistent).

It could be changed to `For a secure alternative, use pinentry or let openfortivpn prompt for the password.` But perhaps that becomes too complex, and we need to keep it simple. Your opinion?